modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -438,13 +416,19 @@ export default class SelectBase extends React.Component<SelectBaseProps> {
           <SelectMenu
             aria-activedescendant={options[focusedOptionIndex].id}
             aria-labelledby={ariaLabelledBy}
+            buttonRef={buttonRef}
             id={this.id}
             error={error}
             isAnimated={isMenuAnimated}
+            isAutoFlipped={isMenuAutoFlipped}
+            isAutoFocused={isMenuAutoFocused}
+            isFlipped={isMenuFlipped}
+            isHidden={isMenuHidden}
             isHiding={isMenuHiding}
-            menuRef={menuRef}
+            menuRef={this.menuRef}
             onBlur={onMenuBlur}
             onKeyDown={onKeyDown}
+            onCloseOnEscape={onMenuCloseOnEscape}
           >
```

Comment:
This is starting to be a lot of config options

Reply:
Agreed, the visual testing props are a contributor. Will look into this in the future.